documentation/docs/verifiable_credentials/verifiable_presentations.mdx

```diff
@@ -10,17 +10,157 @@ keywords:
+
+Of the above, only the list of credentials is required when creating a presentation using the framework.
+However, the holder property should be included to satisfy [subject-holder relationship](#subject-holder-relationship) checks during validation.
+
+After creation, the holder signs the verifiable presentation using a private key linked to one of the verification methods in their
+DID Document and transmits it to a verifier for [validation](#validation).
+
+### Proof Options {#proof-options}
+
+A digital signature on a verifiable presentation both provides data integrity and proves the DID of the holder.
+The proof section embedded in a presentation may also include additional metadata.
+
+The following metadata properties can be configured by the framework and are optional and omitted by default:
+
+- **Created**: timestamp of when the presentation was signed.
+- **Expires**: timestamp after which the presentation is no longer considered valid.
+- **Challenge**: arbitrary random string. Sent by the verifier and mitigates replay attacks; should be sufficiently random and uniquely generated per presentation request.
+- **Domain**: arbitrary string. Sent by the verifier and can help mitigate replay attacks when used with a challenge.
+- **Proof Purpose**: indicates the purpose of the signature.
+  - **AssertionMethod**: to assert a claim. The signing verification method must have an [`assertionMethod`](https://www.w3.org/TR/did-core/#assertion) relationship to be valid.
+  - **Authentication**: to authenticate the signer. The signing verification method must have an [`authentication`](https://www.w3.org/TR/did-core/#authentication) relationship to be valid.
+
+Verifiers should always send a challenge and domain to mitigate replay attacks, see [Security Considerations](#security-considerations).
+
+A verifier could also choose to ignore some or all of these options.
+See [Proof Verifier Options](#proof-verifier-options) for more information.
+
+## Validation {#validation}
+
+The IOTA Identity Framework provides several options for verifiers to validate various sections of a verifiable presentation.
+See the [example](#example) for a demonstration of how to validate a presentation.
+
+The framework checks:
+
+- **Semantic structure**: ensures the presentation and its credentials adhere to the specification.
+- **Presentation proof**: verifies the presentation signature against the DID Document of the holder.
+- **Credential proofs**: verifies the credential signatures against the DID Documents of their respective issuers.
+- **Optional validations**: additional checks on signatures and credential fields can be configured by the verifier.
+
+Note that a verifier may specify which DID Documents to use for the holder and issuers, otherwise they are resolved from the IOTA Tangle automatically.
```

Comment:
I'd recommend "from the Tangle" instead of "IOTA Tangle", since it would also work on non IOTA tangles such as private networks.